runtime/src/bank.rs

```diff
@@ -5131,6 +5135,84 @@ pub(crate) mod tests {
         bank
     }
 
+    fn assert_capitalization_diff(bank: &Bank, updater: impl Fn(), asserter: impl Fn(u64, u64)) {
+        let old = bank.capitalization();
+        updater();
+        let new = bank.capitalization();
+        asserter(old, new);
+        assert_eq!(bank.capitalization(), bank.calculate_capitalization());
+    }
+
+    #[test]
+    fn test_store_account_and_update_capitalization_missing() {
```

Comment:
classical textbookish non-overlapping exhaustive shortly-chucked set of unit tests for `store_account_and_update_capitalization`, which is newly promoted from `cfg(test)`.